README.md

```diff
@@ -4,6 +4,70 @@ dice-fairlink is a JDBC driver designed to connect to the read replicas of an AW
 The driver will periodically obtain a description of the cluster and dispatch connections to each read replica
 on round-robin fashion.
 
+# Why do we need dice-fairlink (TL/DR version)?
+Because in many cases Aurora will not evenly distribute the connections amongst all the available read replicas.
```

Comment:
we probably should say why the links are not distributed properly by aurora.

Reply:
It's explained elsewhere (not on the TL/DR section)